src/packages/v4/components/V4SplitList.tsx

```diff
@@ -0,0 +1,24 @@
+import { BigNumber } from "@ethersproject/bignumber"
+import SplitList, { V2V3SplitListProps } from "packages/v2v3/components/shared/SplitList"
+import { V4Split } from "../models/v4Split"
+import { v4SplitToV2V3Split } from "../utils/v4Splits"
+
+type V4SplitListProps = Omit<V2V3SplitListProps, 'splits' | 'currency' | 'totalValue'> & {
```

Comment:
Hmm. I think we should keep v4 totally discinct from v3. WDYT about just recreating the type script scratch here?

Reply:
On second look it's definitely a good call. I copied all the SplitsList and SplitItem components over to the V4 package and adjusted for V4 👍